### PR TITLE
Konstantinos / TRANSLATION => [ROW] &&  [EU] Review [Landing/Stocks] page

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [x] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/stocks/_footerBanner.tsx [hunk 1]
```diff
@@ -175,7 +179,7 @@ const FooterBanner = ({
                             <StyledLinkButton
                                 id="dm-footerbanner-signup-1"
                                 type="submit"
-                                secondary
+                                secondary="true"
```
Comment (michio-rework) [inline src/pages/landing/stocks/_footerBanner.tsx:182]: please update this to `boolean`, :pray:

File: src/pages/landing/stocks/_headerSection.tsx [hunk 2]
```diff
@@ -270,11 +270,15 @@ const HeaderSection = () => {
...
                                 {localize(
-                                    'Trade global stocks and stock indices now on our DMT5 platform',
+                                    'Trade global stocks and stock indices now on our Deriv MT5 platform',
                                 )}
                             </StyledHeaderSmall>
                             <BtnDiv>
-                                <StyledLinkButton type="submit" secondary to={'/landing/signup/'}>
+                                <StyledLinkButton
+                                    type="submit"
+                                    secondary="true"
```
Comment (michio-rework) [inline src/pages/landing/stocks/_headerSection.tsx:279]: please change `secondary` to `boolean`, thanks :pray:

**Review by michio-rework — DISMISSED** [review]

File: src/pages/landing/stocks/_footerBanner.tsx (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/pages/landing/stocks/_footerBanner.tsx:182]: Oh I see, thanks

**Comment by michio-rework** [comment]
please resolve conflicts and check the comment :pray:

**Review by habib-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**